data-collection/src/test/java/refactoringml/ProcessMetricTrackerTest.java

```diff
@@ -151,9 +152,31 @@ public void countBugFixes() {
 				qty++;
 			}
 
-			pm.reportCommit("bla bla " + (keywordHere) + "ble ble","Rafael", 10, 20);
+			pm.reportCommit("#" + (i + 2), "bla bla " + (keywordHere) + "ble ble","Rafael", 10, 20);
 		}
 
 		Assert.assertEquals(qty, pm.getCurrentProcessMetrics().bugFixCount);
 	}
+
+	@Test
```

Comment:
It is hard for me to review if the algorithm actually works, so I'm trusting on the (integration) tests. But maybe unit tests focused on the `calculateStability()` and `isStable()` will help us in getting more confidence.